src/sentry/static/sentry/app/views/releases/releaseCommits.jsx

```diff
@@ -18,6 +18,9 @@ const ReleaseCommit = React.createClass({
 
   },
   render() {
+    let name = this.props.repository.name;
+    let match = name.match(/:(.+)/);
```

Comment:
It's probably safer if this match failed gracefully if the `provider:` prefix was absent. (If there is no provider, this `match` call will return `null` and `match[1]` will throw an exception.

I'd do:

```javascript
let match = name.match(/[^:]+$/);
let repoPath = match[0];
```

Results:

```javascript
'github:getsentry/sentry'.match(/[^:]+$/);
// getsentry/sentry
'getsentry/sentry'.match(/[^:]+$/);
// getsentry/sentry
':getsentry/sentry'.match(/[^:]+$/);
// getsentry/sentry
```